Move wiki documents to new style/ directory

nomicode opened the pull request:
the wiki pages should be deleted when this PR is merged

nomicode committed:
Move wiki documents to new style/ directory

matthijskrul commented on style/rst.rst:
```diff
@@ -0,0 +1,464 @@
+these guidelines is required for contributions to our documentation. For ease
+of use, we have gathered them in this wiki page below.
+
+**Note**:
```
This should no longer be here! This was only relevant for Github wiki

matthijskrul commented on style/rst.rst:
```diff
@@ -0,0 +1,464 @@
+wrapped as closely to 79 characters per line as possible, without exceeding
+that limit.
+
+**Tip**:
```
We can also now use Sphinx theme things for tips, warnings, etc again (I just used bold for the wiki)

nomicode committed:
fixup! Move wiki documents to new style/ directory

nomicode commented:
@matthijskrul addressed your comments!

looks like GitHub RST viewing doesn't properly support admonition styles, which sucks. but I'm happy with this as an interim solution

matthijskrul commented:
> @matthijskrul addressed your comments!
> 
> looks like GitHub RST viewing doesn't properly support admonition styles, which sucks. but I'm happy with this as an interim solution

Yeah it has various limitations, also with certain links and such. So we have to choose between adjusting for that (with bolding or somesuch) or using them anyway with the premise that people will get it and so we could easily run it through Sphinx later on without extra effort. I suspect the latter is the more robust one but it's up to you! I just wanted to flag it.
